test/test_languages/testCAndCPP.py

```diff
@@ -93,17 +93,21 @@ def test_double_slash_within_string(self):
         result = get_cpp_function_list("""int fun(){char *a="\\\\";}""")
         self.assertEqual(1, len(result))
 
-    def test_number_with_thousands_seperator_since_cpp14(self):
+    def test_number_with_thousands_separator_since_cpp14(self):
```

Comment:
This is a minor typo fix, done "en-passant"

Reply:
great